src/admin/components/SearchForm.js

```diff
@@ -56,16 +55,16 @@ export class AdminSearchFormBase extends React.Component {
         <input className="AdminSearchForm-query" type="search" name="q"
                ref={(ref) => { this.searchQuery = ref; }}
                placeholder={_('Search')} defaultValue={query} />
-        <button className="button button-middle" type="submit" title="Enter"
+        <Button className="button-middle" type="submit" title="Enter"
                 ref={(ref) => { this.submitButton = ref; }}
-                onClick={this.handleSearch}>
+                onClick={this.handleSearch} styleName="action">
           {_('Search')}
-        </button>
-        <button className="button button-end button-inverse"
+        </Button>
+        <Button className="button-end" inverse styleName="action"
```

Comment:
Having anything more than a single way to make buttons seems to really lead to inconsistency and also a bit of confusion around making buttons. This way groking for buttons in the code base you'll find only one thing, because often when we make "buttons" we're really just talking style. Looking at the code in complete isolation it can be a bit weird (because it's just a capital letter away from an HTML component) but when you see it's imported from `ui/components` I think it makes more sense than the `className` approach.